src/dispatch.jl

```diff
@@ -484,6 +484,21 @@ function Base.:*(α::Number, A::LinearAlgebra.Symmetric{<:AbstractMutable})
     return LinearAlgebra.Symmetric(B, c)
 end
 
+function Base.:*(
+    α::AbstractMutable,
+    A::LinearAlgebra.Symmetric{<:AbstractMutable},
+)
+    c = LinearAlgebra.sym_uplo(A.uplo)
+    B = c == :U ? _mult_upper(α, A) : _mult_lower(α, A)
+    return LinearAlgebra.Symmetric(B, c)
+end
+
+function Base.:*(α::AbstractMutable, A::LinearAlgebra.Symmetric)
+    c = LinearAlgebra.sym_uplo(A.uplo)
+    B = c == :U ? _mult_upper(α, A) : _mult_lower(α, A)
+    return LinearAlgebra.Symmetric(B, c)
+end
+
```

Comment:
Also let them multiply in the other order.

```suggestion

function Base.:*(
    A::LinearAlgebra.Symmetric{<:AbstractMutable},
    α::AbstractMutable,
)
    return α * A
end

Base.:*(A::LinearAlgebra.Symmetric, α::AbstractMutable) = α * A
```